client/landing/subscriptions/components/settings-popover/site-settings/email-me-new-comments-toggle.tsx

```diff
@@ -0,0 +1,43 @@
+import { ToggleControl as OriginalToggleControl } from '@wordpress/components';
+import { useTranslate } from 'i18n-calypso';
+import PopoverMenuItem from 'calypso/components/popover-menu/item';
+
+// This is a fix to get around the fact that the original ToggleControl component doesn't support the disabled prop.
+// TODO: Remove this when the original ToggleControl component supports the disabled prop.
+const ToggleControl = OriginalToggleControl as React.ComponentType<
+	OriginalToggleControl.Props & {
+		disabled?: boolean;
+	}
+>;
+
+type EmailMeNewCommentsToggleProps = {
+	value: boolean;
+	isUpdating: boolean;
+	onChange: ( value: boolean ) => void;
+};
+
+const EmailMeNewCommentsToggle = ( {
+	value = false,
+	isUpdating = false,
```

Comment:
The `value` and `isUpdating` props are marked as required in the `EmailMeNewCommentsToggleProps` type definition. To me, it is a bit contradicting to mark a prop required and then to provide a default value for the prop. Providing a default value implies to me that a prop may be undefined (optional). However, I do not find this as a "must fix" issue, unless it would have been enforced by our lint rules or TS. 👍

Reply:
That's a good point. Thanks for catching that, Maciej! I could address this in a separate PR so we don't block today's launch.